scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -92,6 +92,10 @@ export function SendItem($q,
             scope.destination_last = {send_to: null, publish: null, duplicate_to: null};
             scope.origItem = angular.extend({}, scope.item);
             scope.subscribersWithPreviewConfigured = [];
+            scope.sendPublishSchedule = appConfig.ui && appConfig.ui.sendPublishSchedule === false
+                ? appConfig.ui.sendPublishSchedule
```

Comment:
`appConfig.ui.sendPublishSchedule` can only be false at this point so using a variable is a bit confusing.

What about using a shorter version? `appConfig?.ui?.sendPublishSchedule ?? true`